assets/blocks/course-outline/lesson-block/use-lesson-preview-status.js

```diff
@@ -0,0 +1,41 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import { COURSE_STATUS_STORE } from '../status-preview/status-store';
+
+/**
+ * @param {Object}  props
+ * @param {Object}  props.attributes           Block attributes
+ * @param {boolean} props.attributes.isExample Ignore for example blocks
+ * @param {string}  props.attributes.title     Block title
+ * @param {string}  props.clientId             Block ID
+ * @return {{setPreviewStatus: Function, previewStatus: string}} Preview status and control function.
```

Comment:
Can we add a description to the documentation?

Reply:
Cleaned up docs and imports in 9238f80